RFCs/004-In-Assembly-Parallel-Execution.md

```diff
@@ -52,12 +51,12 @@ When used at the assembly level, all tests within the assembly will be executed
 When used at the Class level, all tests within the class will be executed serially after the parallel execution of all other tests is completed.
 When used at the Method level, the test method will be executed serially after the parallel execution of all other tests is completed.
 
-This can be specified via a .runsettings as a global override to turn OFF parallelization as follows:
+Finally, just as in-assembly parallel execution can be disabled globally via the .runsettings file. This can be specified via a .runsettings as a global override to turn OFF parallelization as follows:
```

Comment:
Both of these lines seems repetitive to me.

Reply:
Sorry, fixed.